File: 0203/main.py
class ListNode:
    def __init__(self, val=0, next=None):
        self.val = val
        self.next = next

class Solution:
    def removeElements(self, head: ListNode, val: int) -> ListNode:
        res = ListNode()
        header = res
        while head != None:
            if head.val != val:
                res.next = head
                res = res.next
                head = head.next
                continue
            head = head.next
        res.next = None
        return header.next

File: 0203/main.swift
import Foundation

  public class ListNode {
      public var val: Int
      public var next: ListNode?
      public init() { self.val = 0; self.next = nil; }
      public init(_ val: Int) { self.val = val; self.next = nil; }
      public init(_ val: Int, _ next: ListNode?) { self.val = val; self.next = next; }
  }
 
class Solution {
    func removeElements(_ head: ListNode?, _ val: Int) -> ListNode? {
        var head = head
        var res: ListNode? = ListNode()
        let header = res
        while head != nil{
            if head!.val != val{
                res?.next = head
                res = res?.next
                head = head!.next
                continue
            }
            head = head!.next
        }
        res?.next = nil
        return header?.next
    }
}